uftrace.c

```diff
@@ -1003,6 +1009,34 @@ static void free_opts(struct opts *opts)
 }
 
 #ifndef UNIT_TEST
+static void apply_default_opts(int *argc, char ***argv, struct opts *opts)
+{
+	char *basename = "default.opts";
+	char opts_file[PATH_MAX];
+	struct stat stbuf;
+
+	/* this is not to override user given time-filter by default opts */
+	parsing_default_opts = true;
+
+	if (opts->mode != UFTRACE_MODE_RECORD &&
+	    opts->mode != UFTRACE_MODE_LIVE &&
+	    opts->mode != UFTRACE_MODE_RECV) {
```

Comment:
You can just return if the mode is one of the case.

Reply:
done.